cmd/ff-proxy/main.go

```diff
@@ -350,74 +375,37 @@ func main() {
 		// Layering them in this order means that the first thing we'll do when we receive an SSE message
 		// is attempt to refresh the cache, then if that's successful we'll forward the event on to Redis and Pushpin
 		cacheRefresher := cache.NewRefresher(logger)
-		redisForwarder := stream.NewForwarder(
-			logger,
-			stream.NewRedisStream(redisClient),
-			cacheRefresher,
-			stream.WithStreamName("proxy:sse_events"),
-		)
+		redisForwarder := stream.NewForwarder(logger, redisStream, cacheRefresher, stream.WithStreamName(sseStreamTopic))
 		messageHandler = stream.NewForwarder(logger, pushpinStream, redisForwarder)
 	}
 
-	connectedStreams := domain.NewSafeMap()
-
-	// If this isn't a read replica Proxy then we'll want to start up a stream with the client
-	// service and receive events
+	// If this is the 'Writer' proxy then open up a stream to Harness Saas
+	// so the Proxy can be notified of changes
 	if !readReplica {
 		streamURL := fmt.Sprintf("%s/stream", clientService)
 		sseClient := stream.NewSSEClient(logger, streamURL, proxyKey, conf.Token())
 
-		clientServiceStream := stream.NewStream(
+		saasStream := stream.NewStream(
 			logger,
 			"*",
```

Comment:
what does the magic '*' do here?

Reply:
Tbh I'm not 100% sure, we do this in the SDK client so I copied it from there. I think it might be something to do with the pushpin channels? I should probably look into this and see if it is and change it to the channel we care about.